coordinator/src/services/translation/common.ts

```diff
@@ -87,12 +87,14 @@ export function onlyElement<T>(previousValue: T, currentValue: T, currentIndex:
 }
 
 export function wrapInOperationOutcome(message: SpineDirectResponse): fhir.OperationOutcome {
-  const severity = message.statusCode <= 299 ? "information" : "error"
-  const code = message.statusCode <= 299 ? "informational" : "invalid"
-  const operationOutcomeIssue = new fhir.OperationOutcomeIssue(severity, code)
-  operationOutcomeIssue.diagnostics = message.body
-
-  const response = new fhir.OperationOutcome()
-  response.issue = [operationOutcomeIssue]
-  return response
+  const operationOutcomeIssue: fhir.OperationOutcomeIssue = {
```

Comment:
We dont need the type here. Given the they type is already defined for the `issue` property on OperationOutcome, if its wrong the compiler will complain about it there